shacl12-profiling/index.html

```diff
@@ -804,29 +804,106 @@ <h3>Document Conventions</h3>
       </p>
+      <section id="base-profiles">
+        <h3>Base Profiles</h3>
+        <p>
+          Base profiles of SHACL are profiles that correspond to the individual SHACL 1.2 specifications listed in the <a href="#specifications">Specifications</a> section, above.
+        </p>
+        <p>
+          Together, these Base Profiles cover all of SHACL and, when used together in the <a href="#union-profile">Union Profile</a>, they can validate that any graph is valid SHACL 1.2 data.
+        </p>
+      </section>
+      <section id="non-base-profiles">
+        <h3>Non-base Profiles</h3>
+        <p>
+          Non-base profiles of SHACL are profiles that do not correspond directly to a SHACL 1.2 specification. These profiles include supersets of the specification-derived base profiles, such as the <a href="#union-profile">Union Profile</a> defined below, and also other profiles of parts of SHACL, such as <a href="#profile-x">Profile X</a> that contains SHACL elements that, when used for validation, require no more than XXX computational complexity.
+        </p>
+        <section id="union-profile">
+          <h4>Union Profile</h4>
+        </section>
+        <section id="profile-x">
+          <h4>Profile X</h4>
+        </section>
+      </section>
+      <section id="creating-profiles">
+        <h3>Creating other Profiles</h3>
+      </section>
 		</section>
 
 		<section id="profiling-with-shacl">
 			<h2>Profiling with SHACL</h2>
-			<p>Content.</p>
+			<p>
+        This section describes how to profile specifications with SHACL.
+      </p>
+      <section id="profile-parts">
+        <h3>Profile Part Roles</h3>
+        <p>
+          SHACL is a language that can implement constraints and inference rules for RDF data. These two things are not the total set of things that a profile designers may wish to do, for example, they may want to include documentation about why and how the profile was created or provide new model elements - classes, predicates etc. - within an extended schema.
+        </p>
+        <p>
+          Since SHACL cannot be used for all possible profile parts, profile designers must look to specifications outside the <a href="#specifications">SHACL family of specifications</a> for guidance on a profile's total set of parts and how to relate them to one another. [[[dx-prof]]] is recommended for this.
+        </p>
+        <p>
+          Referencing [[[dx-prof]]]'s <a href="https://www.w3.org/TR/dx-prof/#resource-roles-vocab">vocabulary of <em>Resource Role Instances</em></a>, it seems clear that SHACL graphs can perform the role of <a href="https://www.w3.org/TR/dx-prof/#Role:validation">Validation</a> within a profile and perhaps also <a href="https://www.w3.org/TR/dx-prof/#Role:specification">Specification</a>, since SHACL can be used to declare Node and Property Shapes just as OWL can declare Classes and Properties.
```

Comment:
It's unfortunate that dx-prof used the wrong words for most if not all of its roles. `Role:validation` *should be* `Role:validator` or _perhaps_ `Task:validation`, though that would mean broader and deeper changes to dx-prof.
```suggestion
          Referencing [[[dx-prof]]]'s <a href="https://www.w3.org/TR/dx-prof/#resource-roles-vocab">vocabulary of <em>Resource Role Instances</em></a>, it seems clear that SHACL graphs can be applied to the role of <a href="https://www.w3.org/TR/dx-prof/#Role:validation">Validation</a> within a profile and perhaps also to the role of <a href="https://www.w3.org/TR/dx-prof/#Role:specification">Specification</a>, since SHACL can be used to declare Node and Property Shapes just as OWL can be used to declare Classes and Properties.
```

Reply:
> It's unfortunate that dx-prof used the wrong words for most if not all of its roles. `Role:validation` _should be_ `Role:validator` or _perhaps_ `Task:validation`, though that would mean broader and deeper changes to dx-prof.

Yes, I think you are correct.

But, we may have a chance to fix this as I've asked for dx-prof to be added to the list of specifications that the proposed renewed DXWG will update and move to Rec.